AppsDialogflowApp.ts

```diff
@@ -0,0 +1,65 @@
+import {
+    IAppAccessors,
+    IConfigurationExtend,
+    IConfigurationModify,
+    IEnvironmentRead,
+    IHttp,
+    ILogger,
+    IModify,
+    IPersistence,
+    IRead,
+} from '@rocket.chat/apps-engine/definition/accessors';
+import { ApiSecurity, ApiVisibility } from '@rocket.chat/apps-engine/definition/api';
+import { App } from '@rocket.chat/apps-engine/definition/App';
+import { ILivechatEventContext, IPostLivechatAgentAssigned } from '@rocket.chat/apps-engine/definition/livechat';
+import { IMessage, IPostMessageSent } from '@rocket.chat/apps-engine/definition/messages';
+import { IAppInfo } from '@rocket.chat/apps-engine/definition/metadata';
+import { ISetting } from '@rocket.chat/apps-engine/definition/settings';
+import { AppSettings } from './AppSettings';
+import { CloseChat } from './endpoints/CloseChat';
+import { PerformHandover } from './endpoints/PerformHandover';
+import { OnSettingUpdatedHandler } from './handler/OnSettingUpdatedHandler';
+import { PostLivechatAgentAssignedHandler } from './handler/PostLivechatAgentAssignedHandler';
+import { PostMessageSentHandler } from './handler/PostMessageSentHandler';
+
+export class AppsDialogflowApp extends App implements IPostMessageSent, IPostLivechatAgentAssigned {
+    constructor(info: IAppInfo, logger: ILogger, accessors: IAppAccessors) {
+        super(info, logger, accessors);
+    }
+
+    public async initialize(configurationExtend: IConfigurationExtend, environmentRead: IEnvironmentRead): Promise<void> {
+        AppSettings.forEach((setting) => configurationExtend.settings.provideSetting(setting));
+        configurationExtend.api.provideApi({
+            visibility: ApiVisibility.PUBLIC,
+            security: ApiSecurity.UNSECURE,
+            endpoints: [
+                new CloseChat(this),
+                new PerformHandover(this),
+            ],
+        });
+        this.getLogger().log('Apps.Dialogflow App Initialized');
+    }
+
+    public async executePostMessageSent(message: IMessage,
```

Comment:
```suggestion
    public async executePostMessageSent(message: ILivechatMessage, read: IRead, http: IHttp, persistence: IPersistence, modify: IModify): Promise<void> {
        const handler = new PostMessageSentHandler(this, read, http, modify);
        await handler.run(message);
    }
```

Please, reduce the unnecessary line breaks. Besides that, there is a lot of indentation issues, you can set up your editor to fix this automatically.